### ISSUE TITLE
Problem in my network

### ISSUE THREAD

**Issue by ankitpatnala** [tensorflow/graphics#137]
Hi @avneesh-sud ,
I am quite new to software and machine learning world. I have some silly queries.
The architecture in mesh-segmentation-task. Does it have any FCN ? i could only see the tf.keras.CONV1D. Why was no dropout choosed ? Do we have softmax activation function at the output layer ?
My problem:
I am currently working with some models which has varied number of vertices in a mesh.it was working well with small simple use cases. But now after increasing complexity the architecture failed to detect any vertex . It showed each and every vertex as class 0. The other change was i increased the number of input dataset to 4800 from 500. Do you reckon any kind of issue with my procedure ?

_Originally posted by @ankitpatnala in https://github.com/tensorflow/graphics/issues/120#issuecomment-563258008_

**Comment by avneesh-sud** [comment]
Hi @ankitpatnala 

The architecture is based on FeastNet (without the graph pooling U-net layers). The conv1d implements the linear layers, and dropout is not needed. 

The classification loss does have softmax activation in the cross entropy computation, look at loss = tf.losses.sparse_softmax_cross_entropy(...) in the colab demo.
We have trained with larger inputs >10k meshes, and it should converge.

Difficult to debug the issue with your procedure remotely. Investigate your data for possible bugs too.
